storybook/.storybook/main.ts

```diff
@@ -24,13 +24,13 @@ const config: StorybookConfig = {
   },
   staticDirs: ['../public'],
   logLevel: 'debug',
-  async viteFinal(config) {
-    return mergeConfig(config, {
-      resolve: {
-        preserveSymlinks: true,
-      },
-    });
-  },
+  // async viteFinal(config) {
+  //   return mergeConfig(config, {
+  //     resolve: {
+  //       preserveSymlinks: true,
+  //     },
+  //   });
+  // },
```

Comment:
Note: this should be left out of this PR, since these changes are also here: https://github.com/OpenRailAssociation/osrd-ui/pull/721